### PR TITLE
fix: removing extra line breaks at the end of para on public page

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6428

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/public/index.js [hunk 1]
```diff
@@ -13,7 +13,12 @@ export default class IndexController extends Controller {
 
   @computed('model.event.description')
   get htmlSafeDescription() {
-    return htmlSafe(this.model.event.description);
+    const description = String(this.model.event.description);
+    let x = description.length;
+    while (description.substr(0, x).endsWith('<br>')) {
+      x = x - 4;
+    }
+    return htmlSafe(description.substr(0, x));
```
Comment (iamareebjamal) [inline app/controllers/public/index.js:21]: :eyes: ??

**Comment by iamareebjamal** [comment]
Read the last comment on the issue

**Comment by maze-runnar** [comment]
> Read the last comment on the issue

I had read that. Why add truncate, why can't just remove line breaks from end string?

**Comment by iamareebjamal** [comment]
> why can't just remove line breaks from end string?

That's the meaning of truncate. But before saving in the server, not after

**Comment by maze-runnar** [comment]
> But before saving in the server, not after

oh..ok

**Comment by iamareebjamal** [comment]
Use regex to do so

File: app/components/widgets/forms/rich-text-editor.js [hunk 2]
```diff
@@ -63,12 +63,11 @@ export default Component.extend({
       const updateValue = () => {
         debounce(this, () => {
           const value = this.editor.getValue();
-          const description = String(value);
-          let x = description.length;
-          while (description.substr(0, x).endsWith('<br>')) {
-            x = x - 4;
+          let description = String(value);
+          while (description.endsWith('<br>')) {
+            description = description.replace(/<br>$/, '');
```
Comment (iamareebjamal) [inline app/components/widgets/forms/rich-text-editor.js]: That's not what I meant. There should be no loop

Comment (maze-runnar) [reply app/components/widgets/forms/rich-text-editor.js]: removed that in the final commit already.

File: app/components/widgets/forms/rich-text-editor.js [hunk 3]
```diff
@@ -63,7 +63,9 @@ export default Component.extend({
       const updateValue = () => {
         debounce(this, () => {
           const value = this.editor.getValue();
```
Comment (iamareebjamal) [inline app/components/widgets/forms/rich-text-editor.js:65]:
```suggestion
          let value = this.editor.getValue();
```

File: app/components/widgets/forms/rich-text-editor.js [hunk 4]
```diff
@@ -63,7 +63,9 @@ export default Component.extend({
...
-          this.setProperties({ _value: value, value });
+          let description = String(value);
+          description = description.replace(/(<br>)*$/g, '');
+          this.setProperties({ _value: description, value: description });
```
Comment (iamareebjamal) [inline app/components/widgets/forms/rich-text-editor.js:68]:
```suggestion
          value = value.replace(/(<br>)*$/g, '');
          this.setProperties({ _value: value, value });
```

**Comment by iamareebjamal** [comment]
Please check now on null and empty descriptions as well

**Comment by maze-runnar** [comment]
yes it's working.

**Pull request merged**